benchmarks/blind_v8/external_001/oracles/V8-002/oracle.py

```
@@ -0,0 +1,269 @@
+    assert code == 0
+    assert out.getvalue() == expected
+    assert err.getvalue() == ''
+
+
+def test_edge_lineendings_and_whitespace(tmp_path):
+    path = tmp_path / 'lineends.txt'
+    lines = [b'x\n', b'x \n', b'x\r\n', b'x\t\n', b'x\n', b'x \n']
+    path.write_bytes(b''.join(lines))
+    seen = set()
+    uniq = []
+    for l in lines:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)